docs/services.md

```diff
@@ -0,0 +1,101 @@
+
+
+- **Scenario one.** Let’s pretend that you’re an incoming faculty and you have several terabytes of data that you want to bring to Stanford. The DARC Team has this expertise and can deploy several tools from [Globus](/_user_guide/data_transfer/){:target="_blank"} to AWS’s Snowball to get your data loaded into our data ethos.  We will work with the data regardless of security level, including high-risk data.
+
+- **Scenario two.** You want to research the prices of cars and how they fluctuate dependent upon investment into new technology.  The data, from your perspective, only exists with one vendor and you wish to purchase the data. By engaging our team, we can help negotiate the terms and conditions in addition to haggling the best price for the data.
+
+!!! note
+    Did you know the GSB Library and Stanford University Libraries already has a vast collection of data? Chances are we have the data you seek; check with us to find out, it could save you a lot of time and money.
+
+
+## New Projects
+
+If you are a GSB researcher that is interested in starting a new project on the [yens](/_getting_started/yen-servers){:target="_blank"}, please complete and submit DARC's [new storage request](https://darc.stanford.edu/yenstorage){:target="_blank"} form. This form allows you to estimate disk usage, and specify any collaborators that should be added to the shared access list.
+
+!!! important
+    All project collaborators must have active Stanford SUNet IDs and login access to the [yens](/_getting_started/yen-servers){:target="_blank"}. If you are collaborating with Stanford researchers external to the GSB or individuals from other universities please visit our [Collaborators on the Yens](/_policies/collaborators){:target="_blank"} page for sponsorship information.
+
+
+## Feedback
+
+Your feedback is very important to us. Please tell us about any issues you run
+into and successes you have working with DARC.
+
+After the project is complete, we ask that you compute [our feedback form](https://darc.stanford.edu/feedback){:target="_blank"} so we can improve our services and support you better.
+
+If you have any questions, feel free to contact us at: [gsb_darcresearch@stanford.edu](mailto:gsb_darcresearch@stanford.edu){:target="_blank"}
+
+## Other Infrastructure
+
+While the [`yen`](/_getting_started/yen-servers){:target="_blank"} servers may be sufficient for many of the research computing problems faced by researchers at the GSB, there are a number of other resources available both at the GSB and across campus.
+
+### Links to other services
```

Comment:
Let's capitalize all words in the title, please

Reply:
done